crates/matrix-sdk-crypto/src/types/one_time_keys.rs

```diff
@@ -0,0 +1,210 @@
+// Copyright 2021 The Ruma contributors.
```

Comment:
IIRC the Rust project removed all such mentions a few years ago because "[foo project] contributors" is not a legal entity that can hold copyright.

Reply:
I was wondering what I should put there, since it's quite hard to figure out the copyright of certain code in the Ruma repo. Any idea what to put there instead?